pkg/collector/corechecks/cluster/kubernetesapiserver/kubernetes_eventbundle.go

```diff
@@ -64,6 +66,9 @@ func (b *kubernetesEventBundle) formatEvents(taggerInstance tagger.Component) (e
 	readableKey := buildReadableKey(b.involvedObject)
 	tags := getInvolvedObjectTags(b.involvedObject, taggerInstance)
 	tags = append(tags, fmt.Sprintf("source_component:%s", b.component))
+	tags = append(tags, "true_source:kubernetes")
+
+	tags = append(tags, fmt.Sprintf("reporting_controller:%s", b.reportingController))
```

Comment:
**❓ question:** ‏I'm not familiar with the code at all so this might sound dumb but with are not using `getEventSource(b.reportingController, b.component)` here to provide more precision ?

Reply:
This tag is meant to represent the raw values of `reportingController`/`component` that can optionally be used to identify the source. We thought it may be useful for users to have this tag on all events so they are able to filter by specific controller names (since `getEventSource` generalizes and returns only integration names). Most events have the controller name reported in `source_component` but a couple do have it in `reportingController`, so added it here so ensure we cover both options. Let me know if that does not make sense/if you think it would be useful to have the source component added for users that do not choose to enable source detection via the config and I would be happy to change it!